Floofbot/Modules/TagCommands.cs

```diff
@@ -133,39 +127,39 @@ public async Task ListTags([Remainder] string content = null)
         }
 
         [Command("remove")]
-        [Summary("Removes a tag")]
+        [Summary("Removes a tag from the server")]
         [RequireUserPermission(GuildPermission.ManageMessages)]
-        public async Task Remove([Summary("Tag name")] string tag)
+        public async Task Remove([Summary("Tag name")] string tag = null)
         {
-            string tagName = $"{tag.ToLower()}:{Context.Guild.Id}";
-            Tag tagToRemove = _floofDb.Tags.FirstOrDefault(x => x.TagName == tagName);
-            if (tagToRemove != null)
+            if (!string.IsNullOrEmpty(tag))
```

Comment:
I basically wrapped the method in a null/empty check here.
The diff is not displaying the change efficiently